devices/kdesv.js

```diff
@@ -0,0 +1,25 @@
+const fz = require('../converters/fromZigbee');
+const tz = require('../converters/toZigbee');
+const exposes = require('../lib/exposes');
+const reporting = require('../lib/reporting');
+const ota = require('../lib/ota');
+const e = exposes.presets;
+
+module.exports = [
+    {
+        zigbeeModel: ['TWV'],
+        model: 'TWV',
+        vendor: 'KDE',
```

Comment:
The brand seems to be UHome and not KDE?

Reply:
It was pervious name on the site. Yes, you are right, sorry, that was my mistake.

Reply:
I changed vendor field and renamed "file.js" for convenience.